File: 1752. Check if Array Is Sorted and Rotated/solution.py
class Solution(object):
  def check(self, nums):
    """
    :type nums: List[int]
    :rtype: bool
    """
    found = 0
    for index in range(len(nums)):
      if nums[index] > nums[(index + 1) % len(nums)]:
        found += 1
      if found > 1:
        return False
    
    return True

File: 1752. Check if Array Is Sorted and Rotated/solution.java
class Solution {
  public boolean check(int[] nums) {
    int found = 0;
    for (int index = 0; index < nums.length; index++){
      if (nums[index] > nums[(index + 1) % nums.length]){
        found += 1;
      }
      if (found > 1){
        return false;
      }
    }

    return true;
  }
}